src/Microsoft.IdentityModel.Logging/TextWriterEventListener.cs

```diff
@@ -52,35 +54,34 @@ public TextWriterEventListener()
             }
             catch (Exception ex)
             {
-                LogHelper.Throw(string.Format(CultureInfo.InvariantCulture, LogMessages.MIML11001, ex.Message), ex.GetType(), EventLevel.Error, ex.InnerException);
+                LogHelper.LogException<InvalidOperationException>(ex, LogMessages.MIML11001);
+                throw ex;
```

Comment:
Didn't notice that carefully. Why not use ex.GetType() then when calling LogException method.